modules/light-clients/08-wasm/keeper/msg_server.go

```diff
@@ -32,6 +33,22 @@ func (k Keeper) StoreCode(goCtx context.Context, msg *types.MsgStoreCode) (*type
 	}, nil
 }
 
+// RemoveCodeHash defines a rpc handler method for MsgRemoveCodeHash
+func (k Keeper) RemoveCodeHash(goCtx context.Context, msg *types.MsgRemoveCodeHash) (*types.MsgRemoveCodeHashResponse, error) {
+	if k.GetAuthority() != msg.Signer {
+		return nil, errorsmod.Wrapf(ibcerrors.ErrUnauthorized, "expected %s, got %s", k.GetAuthority(), msg.Signer)
+	}
+
+	found := types.HasCodeHash(goCtx, msg.CodeHash)
+
+	err := ibcwasm.CodeHashes.Remove(goCtx, msg.CodeHash)
```

Comment:
looking at the implementation of `Remove` it appears errors if the key is not present, this should mean that we will never be able to return a `found` value of `false`. We would instead error out.

I think we can instead do an early return of `&types.MsgRemoveCodeHashResponse{Found: false}`

Reply:
It doesn't return an error if the key is not present.

Reply:
ah looks like it would only error on a nil key, not a not found one 👍

Reply:
Didn't even error on `nil` when I tried